src/Aquifer.Well.API/Endpoints/Bibles/Get/GetBibleResponse.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Text.Json.Serialization;
+using Aquifer.Data.Enums;
+
+namespace Aquifer.Well.API.Endpoints.Bibles.Get;
+
+public class GetBibleResponse
+{
+    public required int BibleId { get; init; }
+    public required string BibleName { get; init; }
+    public required string BibleAbbreviation { get; init; }
+    public required IReadOnlyList<ResponseBook> Books { get; init; }
+}
+
+public class ResponseBook
```

Comment:
These class names are the same ones used for code generation.  I would prefer either `GetBibleResponseBook` or just `Book`.  Thoughts?